internal/namespaces/namespaces.go

```diff
@@ -20,6 +20,10 @@ type Namespace struct {
 	CreatedAt metav1.Time
 }
 
+func (n Namespace) Namespace() string {
```

Comment:
I am unable to see where this is used.

Reply:
It is used to implement the `NamespacedResource` interface, so that we can use the `auth.FilterResources` func. :slightly_smiling_face: 

https://github.com/epinio/epinio/blob/20f1495fc9d366657765b5083be0c34f96d8ee6b/internal/auth/auth.go#L152-L175

Reply:
:+1: Retracting change request.